src/render/markdown/include.ts

```diff
@@ -176,6 +176,14 @@ export function include(
                 const content = md.render(doc.body, env);
                 return /* HTML */ ` <div>${content}</div> `;
             },
+            alt(tokens: MarkdownIt.Token[], index: number) {
+                const token = tokens[index];
+                const needle = token.info.split(" ")[1];
+                const doc = findDocument(docs, needle);
+                const body = doc ? doc.body : token.content;
+                const content = md.render(body, env);
+                return /* HTML */ `${content}`;
```

Comment:
```suggestion
                return content;
```

Lite oöndigt att wrappa i en template-string och vi har ingen HTML markup som behöver formatering här.